e2e/browserstack.conf.js

```diff
@@ -18,7 +20,7 @@ const nightwatchConfig = {
     'browserstack.local': true,
     'browserstack.debug': true,
     'browserstack.localIdentifier': ('wvtester19234' + process.env.BROWSERSTACK_USER).replace(/[^a-zA-Z0-9]/g, ''),
-    build: 'nightwatch-browserstack',
+    build: 'wv-nightwatch-' + timeStamp,
     applicationCacheEnabled: false,
     webStorageEnabled: false,
     marionette: true
```

Comment:
Random addition, now that browserstack changed their dashboard around our 'Project: Untitled Project' jumped out at me. This should change that:
```suggestion
    marionette: true,
    project: 'Worldview'
```